### ISSUE TITLE
Chameleonize doesn't seem to work on Windows 7, Sublime Text 3 Build 3084

### ISSUE THREAD

**Issue by cryzed** [r3a1ay/flat-chameleon-sublime-theme#3]
I wish this worked on my configuration, but however often I execute the Chameleonize command, the theme doesn't seem to change from its [default colors](http://i.imgur.com/Xdc1qfE.png). There isn't any useful output in the Sublime Text console either, even directly after executing the Chameleonize command there's no info about any reloading of theme files.

I simply installed the theme via Package Control and adjusted the theme-setting in my preferences.

Could you also possibly tell me the name of the color-scheme and font you are using at the very top in this [screenshot](https://raw.githubusercontent.com/r3a1ay/helper/master/img/flat_chameleon/theme.png)? I'm also honestly curious about your font rendering, could you tell me which system you are running Sublime Text on?

**Comment by r3a1ay** [comment]
Please share all console output after `chameleonize` command.

I'm running ST on Linux
Font - `Ubuntu Mono`
You can download color-scheme [here](https://www.dropbox.com/s/1ax42zpwnzj5rlv/Realay%20Red.tmTheme?dl=0). 

To change the font rendering try to experiment with this options (I'm using "subpixel_antialias"):
```js
// Valid options are "no_bold", "no_italic", "no_antialias", "gray_antialias",
// "subpixel_antialias", "no_round" (OS X only), "gdi" (Windows only) and
// "directwrite" (Windows only)
"font_options": ["subpixel_antialias"],
```

**Comment by r3a1ay** [comment]
Please run these commands in sublime's console:
```python
sublime.active_window().active_view().settings().get('color_scheme')
```
```python
sublime.load_settings('Preferences.sublime-settings').get('color_scheme')
```
```python
sublime.load_settings('Preferences.sublime-settings').get('theme')
```

**Comment by cryzed** [comment]
Thank you for sharing! [Here](https://gist.github.com/cryzed/0d4f3e16b9ad084f248b) is the entire console output for a fresh session with an opened Python file and running the Chameleonize command once, in addition to executing the three lines above.

**Comment by r3a1ay** [comment]
Added some fixes.
Please install the theme manually (don't forget to uninstall the theme in package manager).
Show me the console's output, if the issue is still exist.

**Comment by cryzed** [comment]
The issue still exists after manual installation. The log is [here](https://gist.github.com/cryzed/ee8894d712427127298e).

**Comment by r3a1ay** [comment]
I think it should be fixed now.
Please try again.

**Comment by cryzed** [comment]
Not quite unfortunately, I took the liberty of modifying resource.py to add some debug print-statements, I hope [https://gist.github.com/cryzed/4167ca99ad5b2efcf7f1](that) helps.

**Comment by r3a1ay** [comment]
It seems that backslaches should be replaced with slashes in `load_resource` function.
Please try the latest github version.

**Comment by cryzed** [comment]
Yes, I just figured that out myself. I was about to fork the project, I'll test it out immediately. Thank you for your patience.

**Comment by r3a1ay** [comment]
Thanks for your help with this issue.
The fix will be available via package control as v1.3.0 version.
